project/CommonSettingsDisplay.scala

```diff
@@ -0,0 +1,46 @@
+/**
+  * Copyright © 2016-2019 Lightbend, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  *
+  * NO COMMERCIAL SUPPORT OR ANY OTHER FORM OF SUPPORT IS OFFERED ON
+  * THIS SOFTWARE BY LIGHTBEND, Inc.
+  *
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+import sbt.Keys._
+import sbt._
+
+object CommonSettingsDisplay {
+
+   lazy val commonSettings = Seq (
```

Comment:
This object and its member `commonSettings` seem to be defined but not used?

Reply:
It is used for common_display project. Right the line you put next comment

Idea is to have config common for all display subprojects.